pytest_fixtures/broker.py

```diff
@@ -1,12 +1,43 @@
 import pytest
 from broker.broker import VMBroker
+from wait_for import wait_for
 
 from robottelo.constants import BROKER_RHEL77
 from robottelo.hosts import Capsule
 from robottelo.hosts import ContentHost
 from robottelo.hosts import Satellite
 
 
+@pytest.fixture(scope="session")
```

Comment:
What's the benefit of wrapping this function in a fixture? It seems like a regular function could be imported and used just the same.

Reply:
It *might* be slightly more efficient, but that's it. If you want this line gone, I've got no problems with that.

Reply:
fixed